lib/notification_service/notification_service/cli/commands/server_command.py

```diff
@@ -90,22 +97,25 @@ def server_stop(args):
 
     try:
         os.kill(pid, signal.SIGTERM)
-        logger.info("Notification server pid: {} stopped".format(pid))
     except Exception:
         logger.error("Failed to stop Notification server (pid: {}) with SIGTERM. Try to send SIGKILL".format(pid))
         try:
             os.kill(pid, signal.SIGKILL)
-            logger.info("Notification server pid: {} stopped".format(pid))
         except Exception as e:
             raise RuntimeError("Failed to kill Notification server (pid: {}) with SIGKILL.".format(pid)) from e
```

Comment:
If the Notification server is failed to be killed and got this error, what would the user do?

Reply:
TBH, if we reach here. There are not many things we can tell the user to do. If users can interpret the stack trace, they should know what to do or they should search the error.